projects/templates/projects/bundles/bundle.html

```diff
@@ -0,0 +1,71 @@
+{% extends "../base.html" %}
+{% load projects_extras %}
+{% block title %}TAIPAN | Bundle - {{ bd.name }}{% endblock %}
+{% block content %}
+    <h1>{{ bd.name }}</h1>
+    <div class="card">
+        {% with suite=bd|get_suite %}
+        <table class="list zebra">
+            <tr>
+                <th class="left-align">Test Suite</th>
+                <th class="left-align">Description</th>
+                <th class="right-align">Actions</th>
+            </tr>
+            <tr>
+                <td class="left-align">{{ suite.name }}</td>
+                <td class="left-align">{{ suite.description }}</td>
+                <td class="right-align">
+                    <ul>
+                        <li><a href="{% url 'projects:suite_detail' suite.id %}">View</a></li>
+                    </ul>
+                </td>
+            </tr>
+        </table>
+    </div>
+    <div class="card">
+        <h2>Builds</h2>
+        {% with builds=bd|get_builds %}
+        {% if builds|length > 0 %}
+        <table class="list zebra">
+            <tr>
+                <th></th>
+                <th class="left-align">Status</th>
+                <th class="left-align">Description</th>
+                <th class="right-align">Actions</th>
+            </tr>
+            {% for bid in builds %}
```

Comment:
again, `bid` isn't overly helpful. try to use the singular. something like the following is the pythonic way:

```python
for instance in instances:
  instance.do_your_thing()
```